dbt_coves/tasks/generate/sources.py

```diff
@@ -60,13 +67,24 @@ def run(self):
         db = self.config.credentials.database
         schema_names = [schema.upper() for schema in self.get_config_value("schemas")]
 
+        schema_selectors = []
+        for schema_name in schema_names:
+            if "*" in schema_name:
+                schema_selectors.append(schema_name.replace("*", ".*"))
```

Comment:
what happens if the schema_name does not contain '*'?

Reply:
If the schema_name doesn't contain *, it will use the name as it was used before